### PR TITLE
Week3 js homework

### PR DESCRIPTION
**Pull request by danzelo1**

### CODE REVIEW

File: Javascript/homework/responses/3.3-clock-digits.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+*Tip: You may want to add a border inside the `.clock .number` CSS section to help you better visualize what's going on in these next steps.*
+
+Set the `text-align` to `center` so that the numbers stay centered at the top of the clock, no matter the clock's size. You may also want to change the font family or size here, as well as the text color.
+
+At this point your clock should look something like this:
+
+![Screen Shot 2021-01-16 at 2.16.29 AM](/Users/danielleanzelone/Library/Application Support/typora-user-images/Screen Shot 2021-01-16 at 2.16.29 AM.png)
+
+You'll notice that the numbers overlap in one spot and you can't read them.  In the next step we'll use rotation tools to change the positioning of each digit on the clock.
```
Comment (MinyMeep1) [inline Javascript/homework/responses/3.3-clock-digits.md]: No issues with this section.

File: Javascript/homework/responses/3.4-rotation.md [hunk 2]
```diff
@@ -4,5 +4,33 @@
...
+
+`.clock .number11 { --rotation: 330deg; }`
+
+If you refresh your HTML page, you'll see there's no need to change anything for `.clock .number12` since the 12 is sitting at the top of the clock by default.
+
+Now, your clock should look something like this:
+
+![Screen Shot 2021-01-16 at 10.43.07 PM](/Users/danielleanzelone/Library/Application Support/typora-user-images/Screen Shot 2021-01-16 at 10.43.07 PM.png)
+
+*Don't forget to delete the border from `.clock .number` after this step if you included it earlier.*
```
Comment (MinyMeep1) [inline Javascript/homework/responses/3.4-rotation.md]: No issues with 3.4

File: Javascript/homework/responses/3.5-clock-hands.md [hunk 3]
```diff
@@ -4,4 +4,42 @@
 
 #### Task 5: Designing the hands of your clock
 
-Style the hands so that the seconds hand is the longest and thinnest, and the hour hand is shortest and widest. These hands should also be set to rotate given a degree.
\ No newline at end of file
+Inside of your clock div define three divs - these will represent the hands on our clock.  For each div, set the class to "hand [type of hand]" and set the data-* attribute to `data-[type of hand]-hand`.  For example, the div tag for the hour hand would look like:
```
Comment (MinyMeep1) [inline Javascript/homework/responses/3.5-clock-hands.md]: "Inside of your clock div define three divs" I didn't quite understand this line.

**Review by emsesc — CHANGES_REQUESTED** [review]
Same issues as Week 4:
1. README.md file format was [updated](https://github.com/bitprj/BitCamp/tree/master/repo_logistics/sample-camp)
2. More bullet points than paragraph formatting would make it more readable
3. Make sure titles (h1/h4 tags) adhere to what is [here](https://github.com/bitprj/BitCamp/tree/master/repo_logistics)
4. Fix images by uploading them to a /images directory in /responses `/responses/images`

**Review by emsesc — APPROVED** [review]

**Comment by emsesc** [comment]
@danzelo1 Resolve conflicts, but everything else is good

**Pull request merged**